testng-core/src/main/java/org/testng/xml/LaunchSuite.java

```diff
@@ -345,7 +324,7 @@ static class ClassListSuite extends CustomizedSuite {
         final Map<String, String> parameters,
         final String annotationType,
```

Comment:
Could be removed

Reply:
I want to revisit a bunch of classes in this `org.testng.xml` package and perhaps deprecate them or remove them directly based on my analysis. I will do this in a separate PR.